python/py-protobuf3/Portfile

```diff
@@ -44,6 +45,8 @@ platforms       darwin
 
 compiler.cxx_standard  \
                 2011
```

Comment:
I'd recommend bumping the cxx standard to 2014 too, per a comment in one of the github issues you provided.

Reply:
Plus I'm in the middle of testing the same fix, albeit with the standard set to 2014. I can't guarantee it'll work with 2011...

Reply:
It worked fine for protobuf3-cpp. Ryan would have used 2014 if it had really been needed.

Reply:
Makes sense. Looks good to me, let's push this!